src/Serializer/SimpleJsonSerializer.php

```diff
@@ -40,27 +49,13 @@ public function meta(array $meta)
 
     public function paginator(PaginatorInterface $paginator)
     {
-        $currentPage = (int)$paginator->getCurrentPage();
-        $lastPage = (int)$paginator->getLastPage();
-
         $pagination = [
-            'total'        => (int)$paginator->getTotal(),
-            'count'        => (int)$paginator->getCount(),
-            'per_page'     => (int)$paginator->getPerPage(),
-            'current_page' => $currentPage,
-            'total_pages'  => $lastPage,
+            'pageNumber' => (int)$paginator->getCurrentPage(),
```

Comment:
should we allow for these key `names` to be configurable, or will that slow down the transformer pipeline (have to load in dynamic values for each transformation)?

Reply:
Do you mean to store the config of pagination for transformation in the transformer, which is param of the method "transform" at TransformService?

Reply:
I was thinking of storing it in diaclone `config.php` it would apply to all transformers on the response.
If for example, I want custom names on the pagination metadata, could I control that with a config file, or would it be easier/cleaner to just set a custom serializer?

Reply:
I think that this is not a strong need. Yes, you can create custom serializer and use it.
And maybe if you will want to use other paginations in one project, s then you will create custom serializer.
And also using config file for this will make code readable harder.